kernel/names.ml

```diff
@@ -34,16 +34,8 @@ struct
 
   let hash = String.hash
 
-  let warn_invalid_identifier =
-    CWarnings.create ~name:"invalid-identifier" ~category:"parsing"
-                     ~default:CWarnings.Disabled
-                     (fun s -> str s)
-
-  let check_soft ?(warn = true) x =
-    let iter (fatal, x) =
-      if fatal then CErrors.error x else
-        if warn then warn_invalid_identifier x
-    in
+  let check_soft ?(strict=true) x =
```

Comment:
It used to be called `check_ident_soft` but it was renamed to `Id.check_soft` in 67f5c70a (PMP, Dec 2012). So, to be consistent with the qualified name view, maybe `check_valid`?

BTW, I realize that the comments in the mli have to be updated. I'm going to do it.